package.json

```diff
@@ -69,7 +69,7 @@
     "@angular/platform-browser": "^7.2.13",
     "@angular/platform-browser-dynamic": "^7.2.13",
     "@angular/router": "^7.2.13",
-    "@hmcts/ccd-case-ui-toolkit": "4.0.0",
```

Comment:
I guess this needs to sync with master version, if further changes required version would be 4.x.x-description for pre-release version

Reply:
i will rebase both the toolkit and mc branches with master and change the version number